packages/account/src/Components/forms/form-select-field.tsx

```diff
@@ -0,0 +1,82 @@
+import { Autocomplete, SelectNative } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { Field, FieldProps, FormikErrors } from 'formik';
+import React from 'react';
+import { TGetField, TListItem } from '../additional-kyc-info-modal/form-config';
+
+type TProps = TGetField & {
```

Comment:
can you follow the naming standard for this? 
`TComponentName` optional to add the suffix "Props"